docs/rfc/rfc-1.adoc

```diff
@@ -465,7 +466,36 @@ BTC from the wallet to another random wallet and close this wallet.
 
 [[transaction-incentives]]
 === Transaction Incentives
-stub!
+
+Governable Parameters
+- `max_gas_refund_price`: The highest amount of gwei that the gas refund
+  contract will pay out per gas for a refund transaction.
+
+There are 7 main transactions that require incentives:
+
+==== Transactions That Must Be Submitted By Operators
+- Submit distributed key generation results for new wallet creation (weekly).
+- Signal that funds need to be moved between wallets if a <<heartbeat,heartbeat>> fails.
+- Signal that a staker is not performing their duty and should be marked as
+ineligible for rewards (as in the case that they were inactive during
+heartbeats).
```

Comment:
We will have to see how expensive it is. Sometimes the cost of marking someone as ineligible could be higher than what the DAO saves on rewards.

Not opposed to the idea, I think it makes sense, we'll just need to be very careful during the implementation.